src/028/p028.clj

```diff
@@ -0,0 +1,11 @@
+(defn spiral_diag_sum [x]
```

Comment:
I'm not sure, but I think that Clojure uses "kebab-case", like so `spiral-diag-sum`.
Can you check?

Reply:
TBH, didn't even look, just copied the function name from my elixir solution! 😂
But you're right, I'll update it! Thanks

Reply:
Haha, ok 😊